app/controllers/items_controller.rb

```diff
@@ -9,31 +10,72 @@ def new
     @item = Item.new
   end
   def create
-    @item = Item.create(item_params)
+    @item = Item.create(item_params) 
+    @lineitem = LineItem.create(quantity: params.dig(:item, :quantity), item_id:@item.id)
+    @item.recalculate_current_quantity
     redirect_to items_path()
+    
+
   end
   def show
-    @item = Item.find(params[:id])
+  @lineitems = LineItem.all    
```

Comment:
spacing issue, we should only load that item's line_items instead of all line items

Reply:
done